xenctrlext/xenctrlext.ml

```diff
@@ -45,7 +45,7 @@ external get_max_nr_cpus: handle -> int = "stub_xenctrlext_get_max_nr_cpus"
 external domain_set_target: handle -> domid -> domid -> unit = "stub_xenctrlext_domain_set_target"
 
 module Xsrawext = struct
-  open Xenstore
+  (* open Xenstore *)
```

Comment:
did you mean to delete the line?

Reply:
Yes, I did comment it to see what happens and then forgot about it. Thanks

Reply:
Nothing it is actually using the Xsrawext module anyway. I'll probably delete it completely and run another pair of BVT/BST